pkg/messenger/messenger.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package messenger
```

Comment:
where do you plan to put the implementation? perhaps the interface should be in `pkg/common/api/messenger` or similar?

Reply:
implementation - `pkg/didcomm/messenger`
interface - pkg/didcomm/common/messenger

@troyronda @llorllale  Are you ok with that?

Reply:
@soluchok yes